lib/jekyll-github-metadata/repository.rb

```diff
@@ -118,7 +118,8 @@ def owner_public_repositories
           :accept => "application/vnd.github.mercy-preview+json",
         }
         memoize_value :@owner_public_repositories, Value.new("owner_public_repositories", proc do |c|
-          c.list_repos(owner, options).map do |r|
+          repos = c.list_repos(owner, options) || []
```

Comment:
Sorry, yes this change actually was not relevant to this issue. I left it in because it seemed innocent/simple enough. When you don't have a token setup (or hit an API limit) it throws `NoMethodError: undefined method 'map' for false:FalseClass`. This can actually be seen in the AppVeyor CI runs as well: https://ci.appveyor.com/project/jekyll/github-metadata/builds/43615200/job/n59xtdhu8cr1v4fp#L140

If you prefer I could remove this from here and make a separate PR for it?

Reply:
Let's leave this. Resolving.